faq.md

```diff
@@ -7,13 +7,31 @@ css_id: faq
 
 ### **What makes Uptane different from other SOTA security mechanisms?**
 
-
-
+Security problems occur due to accidental disclosures, malicious attacks, disgruntled insiders.  It is not
+a matter of whether a successful attack will occur, but when.  One key feature of a security system is the
+ability to securely recover from an attack.  This means that an update system must have a way to securely 
+recover from a key loss or compromise.  
+
+For example, suppose a nation-state actor steals a signing key and wants to use it to distribute software (as has
+happened before) [cite Iran, etc.].  The update system must provide a way to revoke the current trusted
+information even if the adversary is able to be a man-in-the-middle for future communications.  Uptane 
+is designed to provide strong security in cases like these and is designed so that failures are 
+compartmentalized and limited in scope.  
+
+No other automotive update system has been designed to work in such rigorous situations or has 
+received the public scrutiny of Uptane.  We follow best practice in the security community by having
+wide-scale, public review.  This has been proven to be essential time and time again to ensure a 
+design will hold up against attackers, especially those as strong as nation-state actors.  Furthermore,
+Uptane's design is heavily influenced by the design of TUF, a widely used software update system 
+with a strong track record of usability and security across millions of devices.  As a free and open
+standard, with no cost to use or adopt, Uptane stands alone in the automotive update space.
 
 
 ### **How does Uptane work with other systems and protocols?**
-
-
+Other mechanisms for performing update such as XXX, YYY, ZZZ, are compatible with Uptane. Uptane can use
```

Comment:
:) What would be good candidates here? Either way, this sounds a lot like the text in https://uptane.github.io/deployment-considerations/customizations.html. I think this should probably link to that, and that way we can keep this brief, too.

Reply:
Customizations mentions TLS, OMA Device Management, and Unified Diagnostic Services. There is a separate section also on transport security, but I don't see any mention of specific security strategies/systems there. I do think linking it to the Customizations page makes sense.

Reply:
I'd prefer we namedrop Redbend, Movimento, etc. solutions here.  The point is to make things that outsiders may incorrectly view as our competitors be correctly identified as providing a different service (insecure updates).

Reply:
Don't forget Tesla

Reply:
As this is not my field of expertise, can someone provide the relevant names.

Reply:
Also @JustinCappos where can I find a citation for the nation state reference. I can look it up myself, but I need some keywords to search this.